CHANGELOG.md

```diff
@@ -8,8 +8,9 @@ Currently, this repo is in Prerelease. When it is released, this project will ad
 
 ## [Unreleased]
 
-### Changes
+### Changes:
```

Comment:
We don't use semicolons anywhere else, was this an ESLint addition?

Reply:
no, it was my fat fingers.